code/datums/weather/weather_types/ash_storm.dm

```diff
@@ -69,6 +69,12 @@
 	sound_wo.start()
 	sound_wi.start()
 
+	for(var/area/A in impacted_areas)
+		var/mob/living/simple_animal/hostile/megafauna/bubblegum/B = locate() in A
+		for(var/obj/effect/decal/cleanable/CL in A)
```

Comment:
shift the bubblegum check before this to not waste cpu

Reply:
Good catch. Fixed.